lib/fastifySession.js

```diff
@@ -4,16 +4,22 @@ const fp = require('fastify-plugin')
 const idGenerator = require('./idGenerator')()
 const Store = require('./store')
 const Session = require('./session')
-const cookieSignature = require('cookie-signature')
+let Signer
 
 function session (fastify, options, next) {
+  if (!Signer) {
+    Signer = require('@fastify/cookie').Signer
```

Comment:
Why is this lazy?

Reply:
Because @fastify/cookie is a dev dependency. If i load it directly then it directly throws an error and never reaches the dependency check of fastify-plugin

Reply:
Aha, gotcha. And `Signer` should not be decorated on `fastify`?

Reply:
feels kind of wrong to decorate Signer on fastify